include/rosconsole_bridge/bridge.h

```diff
@@ -46,6 +46,7 @@ class OutputHandlerROS : public console_bridge::OutputHandler
 {
 public:
   OutputHandlerROS(void);
+  ~OutputHandlerROS();
```

Comment:
Interesting. That means, that the new destructor might not become active in not-recompiled code.
However, this doesn't break ABI.